packages/langchain/src/openai/util.ts

```diff
@@ -180,9 +180,6 @@ type RoleAndContent = {
 
 function mapRoleAndContent(baseMessage: BaseMessage): RoleAndContent {
   const role = mapBaseMessageToRole(baseMessage);
-  if (!['system', 'user', 'assistant', 'tool', 'function'].includes(role)) {
```

Comment:
where did this go?

Reply:
I removed it as `mapBaseMessageToRole` already contains this exact logic, therefore I viewed it as redundancy.